### ISSUE TITLE
What should UmbracoApplicationUrl be?

### ISSUE THREAD

**Issue by Jeavon** [umbraco/UmbracoDocs#3897]
Currently, in some places, the v9 documentation suggests that when setting UmbracoApplicationUrl it should include "/umbraco" (as per v8) such as [here ](https://our.umbraco.com/documentation/Reference/V9-Config/WebRoutingSettings/#web-routing) however [here](https://our.umbraco.com/Documentation/Extending/Health-Check/Guides/FixedApplicationUrl#updating-the-json-configuration) it does not. 

Additionally, the [keep-alive setting](https://our.umbraco.com/Documentation/Reference/V9-Config/KeepAliveSettings/) suggests that you can control the merge of umbracoApplicationUrl which I don't think is an option in v9. (this is also present in the [appsettings.template.json](https://github.com/umbraco/Umbraco-CMS/blob/v9/contrib/src/Umbraco.Web.UI/appsettings.template.json#L36)) - this could be backported from [v8 codebase](https://github.com/umbraco/Umbraco-CMS/blob/v8/dev/src/Umbraco.Web/Scheduling/KeepAlive.cs#L56)....

My suspicion is that the documentation should all be updated to remove the "/umbraco" but would like confirmation as it could be a bug in v9 and it's supposed to include "/umbraco".

Pinging @p-m-j @ronaldbarendse @bergmania

**Comment by bergmania** [comment]
Good question... It definitely looks like V9 expects `UmbracoApplicationUrl` to be without "/umbraco", even though it is opposite to V8.

It will be very breaking to change the behavior, so I think we need to update documentation for v9

**Comment by Jeavon** [comment]
@bergmania agreed! 

Also on the keep-alive setting, I think we should change all documentation examples to `~/api/keepalive/ping` and note that no trailing slash cannot be used. I don't see the use case of the template merging that was implemented in v8. However in looking at the keep-alive, I've discovered that currently any value in appsettings.json is ignored, PR to fix that https://github.com/umbraco/Umbraco-CMS/pull/12224
